nautobot/core/api/views.py

```diff
@@ -364,11 +364,7 @@ class AuthenticatedAPIRootView(NautobotAPIVersionMixin, routers.APIRootView):
 
 
 class APIRootView(AuthenticatedAPIRootView):
-    """
-    This is the root of the REST API.
-
-    API endpoints are arranged by app and model name; e.g. `/api/dcim/locations/`.
-    """
+    """This is the root of the REST API. API endpoints are arranged by app and model name; e.g. `/api/dcim/locations/`."""
```

Comment:
why this change? By convention docstrings start with a single sentence on a line by itself.

Reply:
Sorry I was triaging a different issue when I was working on this, will revert